qemu/tests/balloon_check.py

```diff
@@ -80,10 +81,10 @@ def memory_check(self, step, ballooned_mem):
         guest_mem_ratio = self.params.get("guest_mem_ratio")
         if guest_mem_ratio:
             gcompare_threshold = max(gcompare_threshold,
-                                     float(guest_mem_ratio) * self.ori_mem)
+                                     float(guest_mem_ratio) * self.pre_mem)
         # for windows illegal test:set windows guest balloon in (1,100),free
         # memory of the OS shoul be as small as possible.
-        if ballooned_mem >= self.ori_mem - 100:
+        if self.pre_mem + ballooned_mem <= 100:
```

Comment:
Should be pre_mem - ballooned_mem ?

Reply:
expect_mem - pre_mem = ballooned_mem
for illegal test, expect_mem should be <=100